ansible/inventory/sample-k3s/hosts.ini

```diff
@@ -0,0 +1,12 @@
+[controller]
```

Comment:
Are these just sample local IPs or is there a particular reason to use these?

Reply:
yeah they're sample local IPs, no particular reason to use them.